tests/test_copie.py

```diff
@@ -205,3 +208,100 @@ def test_valid_copier_config(_copier_config_file):
     test_check(result, "test_user_dir")
     test_check(result, "test_valid_copier_config")
     assert result.ret == 0
+
+
+class TestInclude:
+    """Tests the '!include' directive supported by copier."""
+
+    def test_working(self, testdir):
```

Comment:
can you rename this test to make it self explicict (like `test_copy_include_file`) ?